pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -74,6 +74,14 @@ var (
 		ic.Platform.Azure.DefaultMachinePlatform.InstanceType = "Standard_D8s_v3"
 	}
 
+	validAcceleratedNetworkingInstanceTypes = func(ic *types.InstallConfig) {
+		ic.Platform.Azure.DefaultMachinePlatform.InstanceType = "Standard_D8s_v3"
+	}
+
+	invalidAcceleratedNetworkingInstanceTypes = func(ic *types.InstallConfig) {
+		ic.Platform.Azure.DefaultMachinePlatform.InstanceType = "Standard_A1_v2"
```

Comment:
Standard_A1_v2  "vCPUsAvailable" value is "1", 
How about change to "Standard_B4ms"

Reply:
This is not supported in the central region.

Reply:
you need create ticket for standardBSFamily for your test env https://devservices.dpp.openshift.com/support/general_request/?template=general_azure_ticket